docs/USAGE.md

```diff
@@ -0,0 +1,311 @@
+   ```
+3. Set the "service name" that better describes your application:
+   ```batch
+   setx OTEL_SERVICE MyServiceName /m
+   ```
+4. Set the endpoint, e.g. OpenTelemetry Collector that will forward
+the trace data:
+   ```batch
+   setx OTEL_TRACE_AGENT_URL http://localhost:9080/v1/trace /m
+   ```
+5. Optionally, enable trace injection in logs:
+   ```batch
+   setx OTEL_LOGS_INJECTION true /m
+   ```
+
+1. Enable instrumentation for the targeted application by setting
+the appropriate __CLR enable profiling__ environment variable.
+You can enable instrumentation at these levels:
+ - For current command session
+ - For a specific Windows Service
+ - For a specific user
+The follow snippet describes how to enable instrumentation for
+the current command session according to the .NET runtime.
+To enable instrumentation at different levels, see
+[this](#enable-instrumentation-at-different-levels) section.
+   - For .NET Framework applications:
+   ```batch
+   set COR_ENABLE_PROFILING=1
+   ```
+   - For .NET Core applications:
+   ```batch
+   set CORECLR_ENABLE_PROFILING=1
+   ```
+8. Restart your application ensuring that all environment variables above are properly
+configured. If you need to check the environment variables for a process use a tool
+like [Process Explorer](https://docs.microsoft.com/en-us/sysinternals/downloads/process-explorer).
+
+#### Enable instrumentation at different levels
+
+Enable instrumentation for a specific Windows service:
+   - For .NET Framework applications:
+   ```batch
+   reg add HKLM\SYSTEM\CurrentControlSet\Services\<ServiceName>\Environment /v COR_ENABLE_PROFILING /d 1
+   ```
+   - For .NET Core applications:
+   ```batch
+   reg add HKLM\SYSTEM\CurrentControlSet\Services\<ServiceName>\Environment /v CORECLR_ENABLE_PROFILING /d 1
+   ```
+
+Enable instrumentation for a specific user:
+   - For .NET Framework applications:
+   ```batch
+   setx /s %COMPUTERNAME% /u <[domain/]user> COR_ENABLE_PROFILING 1
+   ```
+   - For .NET Core applications:
+   ```batch
+   setx /s %COMPUTERNAME% /u <[domain/]user> CORECLR_ENABLE_PROFILING 1
+   ```
+
+## Configure custom instrumentation
+
+You can build upon the provided tracing functionality by modifying and adding
+to automatically generated traces. The OpenTelemetry Tracing library for .NET
+provides and registers an [OpenTracing-compatible](https://github.com/opentracing/opentracing-csharp)
+global tracer you can use.
+
+OpenTracing versions 0.11.0+ are supported and the provided tracer offers a
+complete implementation of the OpenTracing API.
+
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches, you'll be
+able to present a more detailed representation of the logic and functionality
+of your application, clients, and framework.
+
+1. Add the OpenTracing dependency to your project:
+    ```xml
+    <PackageReference Include="OpenTracing" Version="0.12.0" />
```

Comment:
```suggestion
    <PackageReference Include="OpenTracing" Version="0.12.0" />
```

there is a newer version

Reply:
@dszmigielski I see version 0.12.0 being used by Datadog.Trace.OpenTracing.csproj, it shouldn't be a problem if we use 0.12.1 as suggested by @pellared but it is good to confirm. To keep the PR scope: create an issue to validate the version issue.